Sofascore/Sofascore/Views/UIImageView+Extension.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+import UIKit
+
+extension UIImageView {
+    
+    func loadImage(for storage: String, withID id: Int, completion: @escaping (UIImage?) -> Void) {
+        let urlString = "https://academy-backend.sofascore.dev/\(storage)/\(String(id))/image"
```

Comment:
Samo jos da ovaj url dolazi od api client-a. Tocnije da ovdje pozoves njegovu neku novu metodu getImageURL ili tako nekako. Motivacija za to ako se promjeni base url, morat ces ovdje rucn to mijenjat. Mis to se sad ne bi trebalo desit, ali eto da pripazis na takve stvari.